### PR TITLE
Setup-Vuetify

### PR DESCRIPTION
**Pull request by nguyenvinhduyet**
## Change Description 👋

I'm done setting up Vuetify in Our Project

## Demo 📺

**[Loom Video Demo](https://www.loom.com/share/bf3b3fcb096f49969b07601ef58484dc)**

-  image

### CODE REVIEW

**Review by truonghungit — CHANGES_REQUESTED** [review]
Look good @nguyenvinhduyet, 
I've left a few comments so let me know if anything is unclear.

Also please remove this text: This is a guide for you to create a pull request
```
Bring clarity to the code with visual aids:

Screenshots: cmd + shift + 4 (MacOS).

Gifs: GIPHY Capture (free/MacOS).
```

File: README.md [hunk 1]
```diff
@@ -32,23 +32,3 @@ yarn lint
...
-A [CLI Tool](https://github.com/commitizen/cz-cli) that asks a list of pre-defined questions to work out the commit message.
-
-Running `yarn git-cz` or `npx git-cz` will start the CLI tool (make sure to stage your changes first). Simply answer the questions and `commitizen` will automatically generate a commit message that is correctly formatted.
-
-#### 2. Manually
-
-Create commit using your tool of choice (Emacs, CLI, GitKracken, SourceTree). Commits must follow the format [outlined here](https://gist.github.com/stephenparish/9941e89d80e2bc58a153#format-of-the-commit-message).
-
-**Note:**
-When creating commit messages manually, there is a pre-commit hook that will validate you have used the right format. However, the onus falls on you to ensure that you have correctly identified the change e.g. `feat:`, `fix:`, `chore:` etc.
```
Comment (truonghungit) [inline README.md:54]: Please merge the latest code for master into your branch. I added some content in README

File: package-lock.json [hunk 2]
```diff
@@ -0,0 +1,17273 @@
+{
```
Comment (truonghungit) [inline package-lock.json:1]: We are using the yarn as a package manager, so we will use the `yarn.lock` file, Please delete file `package-lock.json`

File: public/index.html [hunk 3]
```diff
@@ -1,11 +1,13 @@
 <!DOCTYPE html>
-<html lang="">
+<html lang="en">
   <head>
     <meta charset="utf-8">
     <meta http-equiv="X-UA-Compatible" content="IE=edge">
     <meta name="viewport" content="width=device-width,initial-scale=1.0">
     <link rel="icon" href="<%= BASE_URL %>favicon.ico">
     <title><%= htmlWebpackPlugin.options.title %></title>
+    <link rel="stylesheet" href="https://fonts.googleapis.com/css?family=Roboto:100,300,400,500,700,900">
```
Comment (truonghungit) [inline public/index.html:9]: good 💯

**Comment by nguyenvinhduyet** [comment]
Thanks for your response. I have done request change of you

File: src/App.vue [hunk 4]
```diff
@@ -1,32 +1,50 @@
...
+        />
+
+        <v-img
+          alt="Vuetify Name"
+          class="shrink mt-1 hidden-sm-and-down"
+          contain
+          min-width="100"
+          src="https://cdn.vuetifyjs.com/images/logos/vuetify-name-dark.png"
+          width="100"
+        />
```
Comment (truonghungit) [inline src/App.vue:21]: We will do not using this code in the future, I know these codes to demo how the Vuetify look like, but please remove it when commit, we will implement this feature in another task, Please keep a simple pull request

File: src/components/HelloWorld.vue [hunk 5]
```diff
@@ -1,154 +1,133 @@
...
+            class="subheading mx-3"
+            target="_blank"
+          >
+            {{ eco.text }}
+          </a>
+        </v-row>
+      </v-col>
+    </v-row>
+  </v-container>
 </template>
```
Comment (truonghungit) [inline src/components/HelloWorld.vue:74]: Same with App.vue

File: src/main.js [hunk 6]
```diff
@@ -3,11 +3,13 @@ import App from "./App.vue";
 import "./registerServiceWorker";
 import router from "./router";
 import store from "./store";
+import vuetify from "./plugins/vuetify";
```
Comment (truonghungit) [inline src/main.js:6]: good 💯

File: src/views/Home.vue [hunk 7]
```diff
@@ -1,16 +1,13 @@
...
-    <img alt="Vue logo" src="../assets/logo.png" />
-    <HelloWorld msg="Welcome to Your Vue.js App" />
-  </div>
+  <hello-world />
 </template>
 
 <script>
-// @ is an alias to /src
-import HelloWorld from "@/components/HelloWorld.vue";
+import HelloWorld from "../components/HelloWorld";
```
Comment (truonghungit) [inline src/views/Home.vue:6]: same thing with app.vue

File: README.md (hunk 1 above)
Comment (nguyenvinhduyet) [reply README.md]: yes, i added some content in README file

File: package-lock.json (hunk 2 above)
Comment (nguyenvinhduyet) [reply package-lock.json]: I deleted package-lock.json

File: src/App.vue (hunk 4 above)
Comment (nguyenvinhduyet) [reply src/App.vue]: i removed that code

File: src/components/HelloWorld.vue (hunk 5 above)
Comment (nguyenvinhduyet) [reply src/components/HelloWorld.vue:5]: Okay ^^, i removed it

**Review by nguyenvinhduyet — COMMENTED** [review]
Check your change and give me response, thank you

File: package-lock.json (hunk 2 above)
Comment (truonghungit) [reply package-lock.json]: @nguyenvinhduyet  💯

**Review by truonghungit — APPROVED** [review]
Cool

**Pull request merged**